### PR TITLE
PAYARA-2560 Redesign set-monitoring-configuration asadmin command

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

File: nucleus/admin/monitor/src/main/java/fish/payara/admin/monitor/cli/SetMonitoringServiceConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+@Service(name = "set-monitoring-service-configuration")
+@PerLookup
+@CommandLock(CommandLock.LockType.NONE)
+@I18n("set.monitoring.service.configuration")
+@ExecuteOn({RuntimeType.DAS, RuntimeType.INSTANCE})
+@TargetType(value = {CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CLUSTERED_INSTANCE, CommandTarget.CONFIG, CommandTarget.DEPLOYMENT_GROUP})
+@RestEndpoints({
+    @RestEndpoint(configBean = Domain.class,
+            opType = RestEndpoint.OpType.POST,
+            path = "set-monitoring-service-configuration",
```
Comment (OndrejM) [inline nucleus/admin/monitor/src/main/java/fish/payara/admin/monitor/cli/SetMonitoringServiceConfiguration.java]: Note to myself: Also this and the get command need documentation

**Comment by OndrejM** [comment]
Please also ensure that `set-monitoring-service-configuration --enabled=true` is applied dynamicaly, in the same way as `set configs.config.server-config.monitoring-service.monitoring-enabled=true`. Also `set-monitoring-service-configuration --amxenabled=true` should start AMX dynamically same as `set configs.config.server-config.amx-configuration.enabled=true` (dynamic stop isn't supported). And the same with `--mbeansEnabled=true`

**Comment by MeroRai** [comment]
@OndrejM all the commands you mentioned above does apply dynamically.

**Comment by MeroRai** [comment]
Jenkins test please

**Comment by MeroRai** [comment]
Jenkins test please

**Pull request merged**